### ISSUE TITLE
New Member / RSVP mail: Improve the subject line to include the team name 

### ISSUE THREAD

**Issue by craigmaloney** [GetTogetherComm/GetTogether#230]
I quite like getting a notification of new users, but I also believe that what we have currently can be improved. Wondering if the subject line could have more information. I manage several teams so "New team members" for three teams all have the same subject. For Meetup the message I get is "New members in Michigan!/usr/group – UNIX/Linux users in Farmington Hills" which gives me more information before opening the email.

**Comment by ianp5a** [comment]
Yes, improving what you see in your inbox is also the subject of this similar issue #218
_"Event Invite: Header and Content to be complete and focused on the event. #218"_
